docs/how-to-guides/themes/create-block-theme.md

```diff
@@ -1,13 +1,10 @@
 # Create a block theme
 
-The purpose of this tutorial is to show how to create a block theme and help theme developers transition to full site editing.
-It is recommended that you first read the [block theme overview](/docs/how-to-guides/themes/block-theme-overview.md).
+The purpose of this tutorial is to show how to create a block theme and help theme developers transition to full site editing. It is recommended that you first read the [block theme overview](/docs/how-to-guides/themes/block-theme-overview.md).
 
 You will learn about the required files, how to combine templates and template parts, how to add presets for global styles, and how to add blocks and export the templates in the site editor.
 
-Full site editing is an experimental feature, and the workflow in this tutorial is likely to change.
-
-This tutorial is up to date with Gutenberg version 11.0.0.
+Block themes are releasing in WordPress 5.9 as part of the full site editing project.
```

Comment:
Block themes work with older WordPress versions if Gutenberg is active, should we include that?

Reply:
I added a line for previously availabe in Gutenberg plugin 11_